examples/dataset_server.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Record the correspondence between atlas datasets and algorithms for statistical algorithm results